sympy/functions/special/tests/test_spec_polynomials.py

```diff
@@ -45,6 +46,14 @@ def test_jacobi():
         jacobi(m, conjugate(a), conjugate(b), conjugate(x))
 
     assert diff(jacobi(n, a, b, x), n) == Derivative(jacobi(n, a, b, x), n)
+    assert str(diff(jacobi(n, a, b, x), a)) == 'Sum((jacobi(n, a, b, x) + '\
+        '(2*_k + a + b + 1)*RisingFactorial(_k + b + 1, -_k + n)*jacobi(_k, a,'\
+        ' b, x)/((-_k + n)*RisingFactorial(_k + a + b + 1, -_k + n)))/(_k + a'\
+        ' + b + n + 1), (_k, 0, n - 1))'
```

Comment:
Can dummy_eq be used here instead of str?

Reply:
Probably. I thought that there should be a better way to do it, but didn't remember.

I'll push the rest with str as well for now, but may consider changing it.